### ISSUE TITLE
Open external links in new tabs

### ISSUE THREAD

**Issue by LoadingByte** [fsr-itse/1327#643]
This needs some discussion.

In my opinion, when the user clicks on an external link, he often doesn't want to leave 1327 but instead quickly check up on the external resource and then come back, i.e. if he clicks a link to a YouTube video in a tutorial document. Therefore, external links should open in a new tab.

In case the discussion results in this not being the preferred behavior, the author of a document should at least be given the option to mark specific links to open in new tabs.

**Comment by janno42** [comment]
External links should be marked as such and always be opened in a new tab.

**Comment by invliD** [comment]
I disagree. If people want to open anything in a new tab, they can ⌘/ctrl-click the link to get that behavior, they shouldn't be forced to have a new window/tab open.

**Comment by LoadingByte** [comment]
Looking at it from the perspective of experienced users like us, having to ctrl-click or using the middle mouse button seems perfectly reasonable. However, most users I got to observe in the past didn't know about this shortcut. They always used the "open in new tab" option presented in the link's context menu, which is cumbersome to say the least. And some of them didn't even know that that exists and thus had to use many presses on the "back" button to get back to the source site after finishing up on the external site.

Hence all of this, I think we should force the opening of a new tab since it's way more user-friendly for the majority of all users and there are—I guess—not many edge cases where finally closing the source site when going to an external site is the desired behavior.

**Comment by xasetl** [comment]
Nope.

**Comment by LoadingByte** [comment]
@xasetl You're not helpful.

What should we do now? We've got two people voting for the change and two who are against it.

**Comment by janno42** [comment]
And the two people in favor are the ones who decide.
External links should be marked as such and always be opened in a new tab.
If that should turn out to bother many users we can always add a setting.
